package.json

```diff
@@ -48,6 +48,7 @@
     "@gnosis.pm/util-contracts": "^2.0.6",
     "@openzeppelin/contracts": "=2.5.1",
     "@truffle/contract": "4.2.15",
+    "solidity-bytes-utils": "0.0.9",
```

Comment:
>"@gnosis.pm/dex-contracts": "^0.4.1",

this will need to be changed to 
"@gnosis.pm/dex-contracts": "^0.4.2", once the dex-contracts package is published